kustomize/v1alpha8/flatcar-sysext/patch-flatcar.yaml

```diff
@@ -166,20 +166,22 @@ spec:
                         [Service]
                         EnvironmentFile=/run/metadata/flatcar
 ---
-apiVersion: infrastructure.cluster.x-k8s.io/v1alpha7
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha8
 kind: OpenStackMachineTemplate
 metadata:
   name: ${CLUSTER_NAME}-md-0
 spec:
   template:
     spec:
-      image: ${FLATCAR_IMAGE_NAME}
+      image:
```

Comment:
Shouldn't this field only be updated on the PR that is adding support for it?

Reply:
yes, :facepalm: